TwitterAPIExchange.php

```diff
@@ -287,6 +287,13 @@ public function performRequest($return = true, $curlOptions = array())
         {
             $curlOptions[CURLOPT_CUSTOMREQUEST] = $this->requestMethod;
         }
+        
+        if (isset($curlOptions[CURLOPT_HTTPHEADER])) {
```

Comment:
Minor thing here dude! Coding-standards-wise let's keep some consistency, so `{` on new lines and a blank line before the `unset`! :) Then, once the code is squashed into 1 commit, I'll merge! Thanks 👍